src/components/victory-axis.jsx

```diff
@@ -193,8 +193,10 @@ class VAxis extends React.Component {
   }
 
   getTickFormat(props) {
-    if (props.tickFormat) {
+    if (props.tickFormat && _.isFunction(props.tickFormat)) {
       return props.tickFormat;
+    } else if (props.tickFormat && _.isArray(props.tickFormat)) {
+      return (x) => props.tickFormat[_.indexOf(this.ticks, x)];
```

Comment:
^ in general, anywhere we have a function that just accepts `x`, we should also pass along the `index` of x, to allow different formatting depending on whether you're at the left/right of the axis (for example). Just at a glance, besides `tickFormat`, `getTextLines` would be another place that could accept it.